uobtheatre/reports/reports.py

```diff
@@ -265,12 +265,12 @@ def run(self):
                     production.name,
                     production.society.id,
                     production.society.name,
-                    sales_breakdown["total_sales"],
-                    sales_breakdown["total_card_sales"],
+                    sales_breakdown["total_payments"],
+                    sales_breakdown["total_card_payments"],
                     sales_breakdown["total_refunds"],
                     sales_breakdown["total_card_refunds"],
-                    sales_breakdown["net_income"],
-                    sales_breakdown["net_card_income"],
+                    sales_breakdown["net_transactions"],
+                    sales_breakdown["net_card_transactions"],
```

Comment:
Can you change the headings on the reports to match for consistency (Lines 219 to 231). `Sales` -> `Payments` and `Income` -> `Transactions`